carbonserver/details.go

```diff
@@ -7,19 +7,13 @@ import (
 	_ "net/http/pprof"
 	"sync/atomic"
 	"time"
-	"unsafe"
 
 	"go.uber.org/zap"
 
 	"github.com/go-graphite/carbonzipper/zipper/httpHeaders"
-	protov2 "github.com/go-graphite/protocol/carbonapi_v2_pb"
 	protov3 "github.com/go-graphite/protocol/carbonapi_v3_pb"
 )
 
-func metricDetailsV2toV3(ptr *map[string]*protov2.MetricDetails) *map[string]*protov3.MetricDetails {
```

Comment:
/cc @Civil : could you please check - was `metricDetailsV2toV3` needed?

Reply:
That's some leftovers from attempts to cheaply convert data from one format to another. So it should be safe to remove.

Reply:
Thanks for checking, @Civil !